modules/virt-troubleshooting-vm-import.adoc

```diff
@@ -87,6 +87,22 @@ Failed to bind volumes: provisioning failed for PVC
 ----
 +
 You must use the NFS storage class. Cinder storage is not supported. link:https://bugzilla.redhat.com/show_bug.cgi?id=1857784[(*BZ#1857784*)]
+
+ifeval::["{HCOVersion}" == "2.4.1"]
+* The following error message is displayed in the *Virtual Machines* tab of the *Virtualization* screen in the {product-title} console if there is no matching {VirtProductName} template:
++
+----
+The virtual machine could not be imported.
+VMTemplateMatchingFailed: Couldn't find matching template
+----
++
+Check the following:
```

Comment:
If this error appear it means that there is no mapping to a common template.

Resolution:
Either set source VM another OS type, that exist in the template mapping (maybe direct here to the related section in this doc), 
(For example if source VM OS is "Other", and this is a Windows2016 VM - Need to set it's OS to such).
or map it to a CNV common template, using a custom template configMap,
or add first a common template, if there is no such, and then add a mapping for it.

Reply:
Was just thinking that we don't really need to put all this info here. I think that the sentence indicating the cause of the error (no matching OCP template) will be enough.

I can't direct the user to the ConfigMap section because we can't use cross-references in modules.